crates/ruff_benchmark/benches/linter.rs

```diff
@@ -78,12 +80,21 @@ fn benchmark_default_rules(criterion: &mut Criterion) {
     benchmark_linter(group, &LinterSettings::default());
 }
 
-fn benchmark_all_rules(criterion: &mut Criterion) {
-    let mut rules: RuleTable = RuleSelector::All.all_rules().collect();
```

Comment:
It seems that the benchmark was already running all preview rules, but the `preview_mode` was set to `Disabled`. Meaning, it doesn't set other preview-only changes to the linter.